templates/firefox-ios.xliff

```diff
@@ -3957,6 +3961,29 @@ Private Tab</source>
       </trans-unit>
     </body>
   </file>
+  <file original="Shared/Supporting Files/en.lproj/ExternalLink.strings" source-language="en-US" target-language="en" datatype="plaintext">
+    <header>
+      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="16.2" build-num="16C5032a"/>
+    </header>
+    <body>
+      <trans-unit id="ExternalLink.ExternalInvalidLinkMessage.v136" xml:space="preserve">
+        <source>The application required to open that link can't be found.</source>
```

Comment:
Linter is catching an error because a typographical apostrophe was not used here.
Please update accordingly ([documentation](https://mozilla-l10n.github.io/documentation/misc/documentation_styleguide.html?highlight=apostrope#typography))

Reply:
The PR was merged! It should be fixed 🙏